packages/devtools_app/lib/src/shared/service.dart

```diff
@@ -24,20 +24,21 @@ Future<VmServiceWrapper> _connectWithSse(
       ? uri.replace(scheme: 'http')
       : uri.replace(scheme: 'https');
   final client = SseClient('$uri');
-  final Stream<String> stream = client.stream?.asBroadcastStream();
+  final Stream<String> stream =
+      client.stream?.asBroadcastStream() as Stream<String>;
```

Comment:
what if `client.stream?.asBroadcastStream()` is null? since `client.stream` can be null, `client.stream?.asBroadcastStream()` may be null. So should this be `Stream<String>?` instead?

Reply:
Right. Force unwrapped `stream`. We invoke `stream.drain()` later in this method. So, we know it is not null.

Reply:
Maybe I'm missing something here, but since we used to call `client.stream?.asBroadcastStream()`, that would suggest that we expect `client.stream` to sometimes be null. So why are we we force unwrapping `client.stream` here if can sometimes be null? In particular, this looks like it would break our app when running on desktop: https://github.com/flutter/devtools/blob/a616832995d13410a4078b506c91927865ff6904/packages/devtools_app/lib/src/config_specific/sse/_fake_sse.dart#L17.